packages/client/package.json

```diff
@@ -8,6 +8,10 @@
     "test": "is-ci \"test:ci\" \"test:watch\"",
     "test:ci": "jest",
     "test:watch": "jest --watch",
+    "buildAndServe": "npm run webpack:production && npm run serve",
+    "serve": "serve ./dist -l 3001 -s",
+    "cypress": "start-server-and-test buildAndServe http://localhost:3001 cypress:gui",
```

Comment:
`start-server-and-test` is not installed!

`npm i -D start-server-and-test`

Reply:
What do you mean? It's already a dev dependency.

Reply:
Ah, sorry! Must have forgot to run `npm i`!